src/main/drivers/resource.c

```diff
@@ -113,4 +113,6 @@ const char * const ownerNames[OWNER_TOTAL_COUNT] = {
     "RX_SPI_EXPRESSLRS_BUSY",
     "SOFTSERIAL_TX",
     "SOFTSERIAL_RX",
+    "LPUART_TX",
```

Comment:
(maybe refactor all lines into designated initializers:

```suggestion
    [OWNER_LPUART_TX] = "LPUART_TX",
```

Reply:
[Further] refactoring can be done in 4.6

Reply:
@ledvinap please approve if you are okay with the changes. We can refactor remaining items after release.